paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/clone/PickSchemaFilesForCloneOperator.java

```diff
@@ -50,18 +50,19 @@
  * Pick the files to be cloned of a table based on the input record. The record type it produce is
  * CloneFileInfo that indicate the information of copy file.
  */
-public class PickFilesForCloneOperator extends AbstractStreamOperator<CloneFileInfo>
+public class PickSchemaFilesForCloneOperator extends AbstractStreamOperator<CloneFileInfo>
```

Comment:
`CopyMetaFilesOperator` (One parallelism)?

In this operator, just copy meta files directly:
1. create table if needed.
2. copy all schema files.
3. copy snapshot file.
4. copy manifest list files.
5. copy index manifest file.
6. copy statistics file.

And then, one link, send index files to one `CopyIndexFilesOperator` (Multiple parallelism).
And another link, send manifest files to `CopyManifestFilesOperator` (Multiple parallelism).